aten/src/ATen/native/cpu/Activation.cpp

```diff
@@ -227,6 +227,45 @@ void hardshrink_backward_cpu_kernel(TensorIterator& iter, Scalar lambd) {
   });
 }
 
+static void leaky_relu_kernel(TensorIterator& iter, Scalar negval_) {
+  AT_DISPATCH_FLOATING_TYPES(iter.dtype(), "leaky_relu_cpu", [&] {
+    using Vec = Vec256<scalar_t>;
+    auto zero_vec = Vec((scalar_t)(0));
+    auto one_vec = Vec((scalar_t)(1));
+    scalar_t negval = negval_.to<scalar_t>();
+    Vec negval_v = Vec(negval);
+    cpu_kernel_vec(
+        iter,
+        [&](scalar_t a) -> scalar_t {
+          return a > scalar_t(0) ? a : a * negval;
+        },
+        [&](Vec a) -> Vec {
+          auto r = Vec::blendv(negval_v, one_vec, a > zero_vec);
+          return a * r;
```

Comment:
All PR looks good overall, but this vectorization might be subject to precision loss. I need some time to double check, or you can introduce tests that will ensure me.

Reply:
@VitalyFedyunin, thanks, can you tell me which one may cause the precision loss? **blendv** or “*”.

Reply:
I guess @VitalyFedyunin meant `*` because of the worry of `x * 1.0 != x`?

Reply:
Aha, everything looks fine.